test/integration/good/code-gen/cpp.expected

```diff
@@ -6447,8 +6447,8 @@ class mother_model : public model_base_crtp<mother_model> {
       indices = std::vector<int>(3, 0);
       
       current_statement__ = 84;
-      indices = stan::math::array_builder<int>().add(2).add(3).add(1)
-          .array();
+      assign(indices, nil_index_list(), stan::math::array_builder<int>()
+        .add(2).add(3).add(1).array(), "assigning variable indices");
```

Comment:
This is the only change in the existing cpp.expected. The rest is due to the added test model.